jupyterhub/spawners.py

```diff
@@ -343,13 +354,22 @@ def get_pod_manifest(self):
 
             ## Finalize the pod configuration
 
-            # Set the notebook container image
-            self.image_spec = self.image
-
             # Set the repository path to the working directory
             self.working_dir = mount_path
             self.notebook_dir = mount_path
 
+            # add git project-specific annotations
+            self.extra_annotations = {
+                RENKU_ANNOTATION_PREFIX + '/namespace':
```

Comment:
maybe the prefix should contain the `/` to avoid matching `renku.iot`

Reply:
good point -- I'll make the change in the next PR